server.js

```diff
@@ -1,33 +1,96 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+
+import goldenGlobesData from './data/golden-globes.json'
+//console.log(goldenGlobesData)
+
+// Define the port the app will run on. Defaults to 8080, but can be 
+// overridden when starting the server. For example: PORT=9000 npm start
+// Initialize a new server
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+// Generic error message
+const ERROR_NOT_FOUND = {error : "Sorry, we can't find what you are looking for."}
+
+// First route here
+app.get('/', (request, response) => {
+  response.send('Hello world again')
+});
+
+// Define the method - GET. Give it a path - 'nominations'. Handle the response: get 
+// all data and send as a json response to client.
+// The url: /nominations
+app.get('/nominations', (request, response) => {
+  response.json(goldenGlobesData)
+});
+
```

Comment:
You could add the `/nominations/:index` and return one object the easy way by using array's indexes like id from the database 😺